### DISCUSSION TITLE
Can't recognize my file and error communicating with OpenAI

### DISCUSSION THREAD

**Discussion by flypde** [Future-House/paper-qa#145] [Q&A]
- **My operator sysytem**: Windows 11
-**My code**: 
```
from paperqa import Docs

path1 = "D:\\testqa.pdf"
citation1 = "Goodfellow, I., Bengio, Y., & Courville, A. (2016). Deep learning. MIT press."

path2 = "D:\\test.pdf"
citation2 = "Mitchell, T. M. (1997). Machine learning. McGraw Hill."

doc_paths = [path1, path2]
doc_citations = [citation1, citation2]

docs = Docs()
for path, citation in zip(doc_paths, doc_citations):
    docs.add(path, citation)

answer = docs.query("what is the main topic of this article?", k=1, max_sources=1)
print(answer.formatted_answer)
```
Then it occurs that:
![image](https://github.com/whitead/paper-qa/assets/131115024/d5356c80-8f5c-4bf9-96ae-ebaab2a93adf)
It seems that it can't recognize my file and I have no idea.
![image](https://github.com/whitead/paper-qa/assets/131115024/664c71a6-79ef-4689-979b-2b089fa63844)

- **What I have done all**: I used `pip install paper-qa` and set OPENAI_API_KEY environment variable to my openai api key.

**I'm a beginner. Can somebody tell me why? Thanks!**

**Comment by tmsgh** [comment]
I ran into the same problem. In my case, running the  Install Certificates.command file fixed this issue. See https://stackoverflow.com/questions/75920597/openai-error-apiconnectionerror-error-communicating-with-openai/76519628#76519628
